test/functional/services/remote/remote.ts

```diff
@@ -128,8 +128,8 @@ export async function RemoteProvider({ getService }: FtrProviderContext) {
       .manage()
       .window()
       .setRect({ width, height });
-    await driver.executeScript('window.sessionStorage.clear();');
-    await driver.executeScript('window.localStorage.clear();');
+    await driver.executeScript('try { window.sessionStorage.clear(); } catch (err) {};');
+    await driver.executeScript('try { window.localStorage.clear(); } catch (err) {};');
```

Comment:
Why are we just swallowing the error, instead of logging it or something?

Reply:
Yes, we basically don't care if it fails. As an edge case, sometimes storage is clean but since we clear it during each `after` via mocha lifecycle it throws an errors. WebDriver does not handle it, so it is up to us to do it.

Reply:
Okey dokey, I would argue that we should least log out, but I'll leave up to you comrade :) Thanks for responding!

Reply:
@LeeDr what is your opinion? The error message is smth like
```Failed to read the 'sessionStorage' property from 'Window': Storage is disabled inside 'data:' URLs.
```

Reply:
@dmlemeshko yeah, that's a good point too though.